### DISCUSSION TITLE
Not canceling VPS recording

### DISCUSSION THREAD

**Discussion by michalMonster** [streamlink/streamlink#3923] [Q&A] [answered]
Hi, is there any way that even after closing the console of the VPS server, recording of livestram continues all the time, because now, when I turn off the consoles, the recording stops?

**Comment by bastimeyer** [comment, marked as answer]
Your question has nothing to do with Streamlink. Logging out from an interactive shell when there are still jobs/processes running in the background or foreground will terminate/kill those jobs/processes.

https://en.wikipedia.org/wiki/Nohup
